templates/layout/region--footer.html.twig

```diff
@@ -1,5 +1,6 @@
 {% embed "@organisms/site-footer/yds-site-footer.twig" with {
   site_footer__border_thickness: '8',
+  site_footer__theme: getThemeSetting('footer_theme'),
```

Comment:
This is exactly what I was picturing so I hope it works for everyone. I just looked and the getSetting() function returns an immutable config object. For us, this is a string, but doesn't technically have to be a string. I wonder if we should cast the return value to a string or check its return type?

Reply:
I don't have a problem with this, but what if it were a numeric or array at some point?